ws/templates/preferences/discounts.html

```diff
@@ -30,20 +30,20 @@ <h2>Other Discounts</h2>
         <a href="https://www.ikonpass.com/en/college"><i class="fas fa-external-link-alt"></i> College Club Program</a>
       </div>
       <div class="col-md-4 col-md-pull-4 col-xs-12 text-center">
-        <small>$639 Base, $889 Base Plus, $959 Full</small>
+        <small>$569 Base, $889 Full</small>
       </div>
     </div>
   </div>
   <div class="panel-body">
     <p>
-        <strong>To receive a discount for the 2024-25 season</strong>, fill out <a href="https://forms.office.com/r/7Tka0stVG3">a brief form</a>.
+        <strong>To receive a discount for the 2025-26 season</strong>, fill out
+        <a href="https://docs.google.com/forms/d/e/1FAIpQLSf0gLZ24mSCX0tM0FiMJS9ZJ6YavXDC5fTrgG3CQJnYzoBG1Q/viewform?usp=sharing">a brief form</a>.
     </p>
     <hr>
     <p>
         MITOC student members can get discounted passes:
-        $959 for the Ikon Pass,
-        $889 for the Ikon Base Plus Pass,
-        and $639 for the Ikon Base Pass.
+        $889 for the Ikon Pass,
```

Comment:
No Oxford comma on a list of two items!

```suggestion
        $889 for the Ikon Pass